api/dp-openapi.yaml

```diff
@@ -1020,6 +1189,69 @@ components:
       items:
         type: string
 
+    proof_page_text:
+      type: object
+      properties:
+        text:
+          description: the page text from current round.
+          type: string
+
+    proof_page_data:
+      type: object
+      properties:
+        text:
+          description: the page text. If page state is 'out' from OCR or previous round, else from current round
+          type: string
+        pagestate:
+          description: the page state, for checking continuity. Either out or temp
+          type: string
+        saved:
+          description: true if the page state is 'temp'
```

Comment:
Will this not also return `true` if the page was saveAsDone -- that also saves the page.

Reply:
It doesn't return anything for saved as done because the PI then goes back to the project page so anything returned would be discarded. But we could return something since the PI doesn't have to work like that.

Reply:
If the page has been saved this should be set to `true` regardless of what the PI currently does or doesn't do. We need to think about a general endpoint user and not just the current PI.

Reply:
Are you saying we should return all this data if the page is saved as done?